app/controllers/middleware_server_group_controller.rb

```diff
@@ -3,12 +3,44 @@ class MiddlewareServerGroupController < ApplicationController
   include Mixins::GenericShowMixin
   include EmsCommon
   include MiddlewareCommonMixin
+  include Mixins::MiddlewareDeploymentsMixin
 
   before_action :check_privileges
   before_action :get_session_data
   after_action :cleanup_action
   after_action :set_session_data
 
+  OPERATIONS = {
+    :middleware_server_group_reload         => {:op  => :reload_middleware_server_group,
+                                                :msg => N_('Reload')
```

Comment:
One more thing -- if that's what it is, then in
```ruby
add_flash(_("%{operation} initiated for selected server(s)") % {:operation => operation_info.fetch(:msg)})
```
you need to apply `_()` on `operation_info.fetch(:msg)` otherwise the operation string
won't be translated.